client/app/components/queries/QueryEditor/index.jsx

```diff
@@ -158,6 +158,8 @@ QueryEditor.propTypes = {
   syntax: PropTypes.string,
   value: PropTypes.string,
   autocompleteEnabled: PropTypes.bool,
+  autoLimitChecked: PropTypes.bool,
+  autoLimitAvailable: PropTypes.bool,
```

Comment:
Those + the default props below can also be removed as they are not used here (and it actually displays an alert in the console)
<img width="473" alt="Screen Shot 2020-09-02 at 18 31 12" src="https://user-images.githubusercontent.com/3356951/92038862-7dabdf00-ed4a-11ea-8b2f-9b0f49f0cce3.png">

Reply:
I think this is necessary because we have to give it a default value as I did in line 173. The warning is because previously I deleted the definition in const QueryEditor, now I added it back and it works fine.

Reply:
@gjxdxh if you remove it from every QueryEditor call it should work as the QueryEditor doesn't make any use of the `autoLimit` information (the QueryEditorControls is a separate component).